sysid-library/src/main/cpp/logging/SysIdLogger.cpp

```diff
@@ -51,6 +52,13 @@ double SysIdLogger::MeasureVoltage(
       if constexpr (frc::RobotBase::IsSimulation()) {
         fmt::print("Recording SPARK MAX voltage\n");
       }
+    } else if (controllerNames[i] == "TalonFX (Pro)") {
+      auto* ctreController =
+          dynamic_cast<ctre::phoenixpro::hardware::TalonFX*>(controller);
+      sum += ctreController->GetDutyCycle().GetValue().value() * 12.0;
```

Comment:
It would be more accurate to record the applied voltage instead of multiplying the duty cycle by 12.